pxr/usd/usdGeom/validators.cpp

```diff
@@ -144,6 +144,36 @@ _SubsetParentIsImageable(const UsdPrim& usdPrim)
     };
 }
 
+static
+UsdValidationErrorVector
+_GetGPrimParentingErrors(const UsdPrim& usdPrim)
+{
+    UsdValidationErrorVector errors;
+
+    if (usdPrim.IsA<UsdGeomBoundable>()){
+        UsdPrim parentPrim = usdPrim.GetParent();
+        while (parentPrim && !parentPrim.IsPseudoRoot()) {
+            if (parentPrim.IsA<UsdGeomGprim>()) {
+                errors.emplace_back(
+                    UsdValidationErrorType::Error,
+                    UsdValidationErrorSites{
+                            UsdValidationErrorSite(usdPrim.GetStage(),
+                                                   usdPrim.GetPath())
+                    },
+                    TfStringPrintf("Gprim <%s> has an ancestor prim that is also a Gprim,"
```

Comment:
Should this be `Boundable <%s> has an ancestor prim that is a Gprim`?

Reply:
Agreed, updated